src/lib/vortex/VoiceClient.ts

```diff
@@ -62,11 +64,13 @@ export default class VoiceClient extends EventEmitter<VoiceEvents> {
                     case WSEventType.UserJoined: {
                         this.participants.set(data.id, {});
                         this.emit("userJoined", data.id);
+                        playSound("call_join");
                         break;
                     }
                     case WSEventType.UserLeft: {
                         this.participants.delete(data.id);
                         this.emit("userLeft", data.id);
+                        playSound("call_leave");
```

Comment:
This will not respect the user's sound settings.
This can be solved by using the `SoundContext`.. which is not available here.
I have previously had the sound, but the audio was really broken on most devices, not sure why.